packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -44,14 +58,27 @@ class WebFlutterDriver extends FlutterDriver {
   @override
   async_io.WebDriver get webDriver => _connection._driver;
 
+  /// Whether to print communication between host and app to `stdout`.
+  final bool _printCommunication;
+
+  /// Whether to log communication between host and app to `flutter_driver_commands.log`.
+  final bool _logCommunicationToFile;
+
   /// Creates a driver that uses a connection provided by the given
   /// [hostUrl] which would fallback to environment variable VM_SERVICE_URL.
   /// Driver also depends on environment variables DRIVER_SESSION_ID,
   /// BROWSER_SUPPORTS_TIMELINE, DRIVER_SESSION_URI, DRIVER_SESSION_SPEC,
   /// DRIVER_SESSION_CAPABILITIES and ANDROID_CHROME_ON_EMULATOR for
   /// configurations.
+  ///
+  /// See [FlutterDriver.connect] for more documentation.
   static Future<FlutterDriver> connectWeb(
-      {String? hostUrl, Duration? timeout}) async {
```

Comment:
The formatting for this is a little odd. Please look into some other examples of how we format named parameter lists, usually something like:

```dart
Foo({
  Bar bar,
  Fizz fizz,
}) async { 
  // ....
}

```

Reply:
i updated. thanks you.

Reply:
Looks good!